backend/src/main/java/com/morjo/model/service/QuizService.java

```diff
@@ -44,4 +47,18 @@ public long createQuiz(Quiz quiz) {
 
         return quiz.getQuizId();
     }
+
+    public int submitQuizResult(QuizSubmit quizSubmit) {
+        long quizId = quizSubmit.getQuizId();
+        Quiz quiz = quizDao.selectQuizById(quizId);
+
+        if ((quizSubmit.getUserAnswer() < 1 && quizSubmit.getUserAnswer() > 4)
```

Comment:
또는 으로 하는게 맞을거같은데 맞나?
```suggestion
        if (quizSubmit.getUserAnswer() < 1 || quizSubmit.getUserAnswer() > 4
```

Reply:
아 맞지 바로 수정할게요